android/app/src/main/java/org/fossasia/openevent/adapters/SponsorsListAdapter.java

```diff
@@ -203,6 +204,9 @@ public int getItemViewType(int position) {
         @BindView(R.id.sponsor_type)
         TextView sponsorType;
 
+        @BindView(R.id.sponsor_name)
```

Comment:
@arp95 
I think it's ok, codacy gives this issue with butterknife.
This isn't an issue as such.